cypress/cypress/e2e/EngageSphere_API.cy.js

```diff
@@ -0,0 +1,62 @@
+// Recupera clientes com sucesso (por exemplo, verifica o código de status 200)
+describe('Recuperação de Clientes', () => {
+    it('Deve recuperar clientes com sucesso', () => {
+      cy.request('GET', 'http://localhost:3001/customers?page=2&limit=10&size=All')
+        .then((response) => {
```

Comment:
Use desestruturação de objetos.

Veja abaixo:

```suggestion
        .then(({ status }) => {
```

> O mesmo vale para os outros testes.